spk/owncloud/Makefile

```
@@ -1,44 +1,49 @@
 SPK_NAME = owncloud
-SPK_VERS = 8.1.12
-SPK_REV = 8
+SPK_VERS = 10.12
+SPK_REV = 9
 SPK_ICON = src/owncloud.png
 DSM_UI_DIR = app
 
-DEPENDS = cross/busybox cross/$(SPK_NAME)
+DEPENDS = cross/$(SPK_NAME)
+SPK_DEPENDS = "WebStation:PHP7.4:Apache2.4"
+
+REQUIRED_MIN_DSM = 6.0
 
 MAINTAINER = ymartin59
 DESCRIPTION = ownCloud is a personal cloud which runs on your own server and gives you freedom and control over your own data.
-ADMIN_URL = /owncloud/
+CHANGELOG = "1. Update ownCloud to 10.12.<br/>2. Use faster SQLite database.<br.>3. Use shared folder for data directory."
```

I believe there is a typo in your change log.

```suggestion
CHANGELOG = "1. Update ownCloud to 10.12.<br/>2. Use faster SQLite database.<br/>3. Use shared folder for data directory."
```

---

Yes, you are correct. Fixed in my local copy and will commit once all the other comments are in.